### DISCUSSION TITLE
AttributeError: 'Whisper' object has no attribute 'transscribe'

### DISCUSSION THREAD

**Discussion by ghost** [openai/whisper#892] [Q&A]
Hi,
Try this python code after setup

model = whisper.load_model("base")
result = model.transscribe("file.mp3")
print(result["text"])

and get this error: "AttributeError: 'Whisper' object has no attribute 'transscribe'"

I install whisper using the command "pip install git+https://github.com/openai/whisper.git"
I also try this code 

import whisper 
print(whisper.__file__)

and get the path ends with \whisper\__init__.py

How can I solve this problem?

**Comment by Jeronymous** [comment]
There is a typo: one "s" in transscribe

**Reply by ghost** [reply]
I screwed up
thank you for your reply
